### PR TITLE
Metadata container to enable metadata tracking

### PR DESCRIPTION
**Pull request by hongye-sun**
The container uses ml-metadata lib to track container execution. 

This PR is not the final state. The following will be added into the PR:
1. track artifacts
2. setup the right db connection

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1486)

### CODE REVIEW

**Comment by gaoning777** [comment]
Are we still merging this or dropping it.

**Comment by neuromage** [comment]
Let's not merge it just yet.

File: components/metadata/container/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.6-slim
+
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q wget
+
+RUN pip install fire==0.1.3 
+RUN pip install tensorflow==1.13.1
```
Comment (Ark-kun) [inline components/metadata/container/Dockerfile]: Maybe we should just start with the `tensorflow/tensorflow:1.13.1-py3` image.

File: components/metadata/container/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.6-slim
+
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q wget
```
Comment (Ark-kun) [inline components/metadata/container/Dockerfile]: Do we use `wget`?

**Pull request closed without merging**